packages/network/lib/cors.ts

```diff
@@ -43,46 +54,33 @@ function parseUrlIntoDomainTldPort (str) {
     }
   }
 
-  const obj = {}
+  const obj: ParsedHost = {}
 
   obj.port = port
   obj.tld = parsed.tld
   obj.domain = parsed.domain
-  // obj.protocol = protocol
 
   debug('Parsed URL %o', obj)
 
   return obj
 }
 
-function urlMatchesOriginPolicyProps (urlStr, props) {
+export function urlMatchesOriginPolicyProps (urlStr, props) {
   // take a shortcut here in the case
   // where remoteHostAndPort is null
   if (!props) {
     return false
   }
 
-  const parsedUrl = this.parseUrlIntoDomainTldPort(urlStr)
+  const parsedUrl = parseUrlIntoDomainTldPort(urlStr)
 
   // does the parsedUrl match the parsedHost?
   return _.isEqual(parsedUrl, props)
 }
 
-function urlMatchesOriginProtectionSpace (urlStr, origin) {
+export function urlMatchesOriginProtectionSpace (urlStr, origin) {
   const normalizedUrl = uri.addDefaultPort(urlStr).format()
   const normalizedOrigin = uri.addDefaultPort(origin).format()
 
   return _.startsWith(normalizedUrl, normalizedOrigin)
 }
-
-module.exports = {
-  parseUrlIntoDomainTldPort,
-
-  parseDomain: _parseDomain,
-
-  getSuperDomain,
-
-  urlMatchesOriginPolicyProps,
-
-  urlMatchesOriginProtectionSpace,
-}
```

Comment:
Is exporting per function the recommended pattern we use at Cypress for exports? I thought we typically used modele.exports at the bottom with the listed functions? Is this a TS thing? You can't do it this way anymore?

Reply:
Yeah, it's better to use the `export` way of doing it I think, but that wasn't available because this was a `.js` file previously.